src/components/List/ListItem.module.css

```diff
@@ -0,0 +1,10 @@
+.list-item{
+    --component-list_item-border_bottom_color: #BCC7CC;
+    --component-list_item-border-bottom-style: solid;
+    --component-list_item-border-bottom-width: 0.1rem;
+    border-bottom-color: var(--component-list_item-border_bottom_color);
+    border-bottom-style: var(--component-list_item-border-bottom-style);
+    border-bottom-width: var(--component-list_item-border-bottom-width);
+    min-height: 4.5rem;
```

Comment:
Do we need a min-height? We've tried to avoid adding this for the other components we have